projects/default/controllers/ros/RosTouchSensor.cpp

```diff
@@ -19,8 +19,11 @@
 
 RosTouchSensor::RosTouchSensor(TouchSensor *touchSensor, Ros *ros) : RosSensor(touchSensor->getName(), touchSensor, ros) {
   mTouchSensor = touchSensor;
-  mTypeServer = RosDevice::rosAdvertiseService((ros->name()) + '/' + RosDevice::fixedDeviceName() + "/get_type",
-                                               &RosTouchSensor::getTypeCallback);
+
+  RosDevice::rosAdvertiseService((ros->name()) + '/' + RosDevice::fixedDeviceName() + "/get_type",
```

Comment:
Is there a reason for modifying this line?

Reply:
No, it is my mistake